src/handlers.js

```diff
@@ -236,17 +277,21 @@ async function jobHandler(message) {
     });
     if (graphConfig.tasks.length) {
       let taskGroupId = graphConfig.tasks[0].task.taskGroupId;
+      debug('Creating queue...');
       let queue = new taskcluster.Queue({
         baseUrl: context.cfg.taskcluster.queueBaseUrl,
         credentials: context.cfg.taskcluster.credentials,
         authorizedScopes: graphConfig.scopes,
       });
+      debug('Creating tasks...');
       await Promise.all(graphConfig.tasks.map(t => queue.createTask(t.taskId, t.task)));
 
       // We used to comment on every commit, but setting the status
       // is a nicer thing to do instead. It contains all of the same
       // information.
-      await context.github.repos.createStatus({
+
+      debug('Trying to create status...');
```

Comment:
Since we had some 404 errors with this call, can you add owner, repo, and sha to this log message?